### DISCUSSION TITLE
where can i get yarp log like "yInfo()"

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#640] [Q&A] [answered]
My code executed normally, and my question is that where can i get the log info as below, i didn't find it in neither yarpserver nor yarpmanager terminal.

`yInfo()<<"Yielding new target: "<<target<<" [deg]";`

**Comment by pattacini** [comment, marked as answer]
Hi @jimingda 

You can find the info printed in the console from which you launched your module or within the `yarplogger` GUI.

Here's an example of instantiation of the `yarplogger` within a `yarpmanager` app:
- https://github.com/vvv-school/tutorial_joint-interface/blob/master/app/scripts/tutorial_joint-interface-system.xml#L5-L9

At any rate, the robot's apps are already configured in terms of `yarplogger`.
